Example/STKWebKitViewController.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "STKWebKitViewController/STKWebKitViewController-Prefix.pch";
 				INFOPLIST_FILE = "STKWebKitViewController/STKWebKitViewController-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
I did this because the 1password extension currently doesn't support targets < 8.0. I have a pull request in with them to fix this. https://github.com/AgileBits/onepassword-app-extension/pull/161

I could point the 1password pod to my repo that supports this and put the deployment target back to 7.0.

I guess you have this set to 7.0 so that apps can use this if they have detected that the phone is on iOS 8.0?